File: java/src/main/java/deltix/luminary/NameValuePair.java
package deltix.luminary;

public class NameValuePair {
    private final String name;
    private final Literal value;

    public NameValuePair(String name, Literal value) {
        this.name = name;
        this.value = value;
    }

    public String getName() {
        return name;
    }

    public Literal getValue() {
        return value;
    }
}

File: csharp/main/NameValuePair.cs
using System;

namespace Deltix.Luminary
{
	public class NameValuePair
	{
		public NameValuePair(String name, Literal value)
		{
			Name = name;
			Value = value;
		}

		public String Name { get; }

		public Literal Value { get; }
	}
}
